doc/source/user_guide/timedeltas.rst

```diff
@@ -53,10 +53,6 @@ You can construct a ``Timedelta`` scalar through various arguments:
    pd.Timedelta('P0DT0H1M0S')
    pd.Timedelta('P0DT0H0M0.000000123S')
 
-.. versionadded:: 0.23.0
-
-   Added constructor for `ISO 8601 Duration`_ strings
```

Comment:
might be with adding this ref in the prose

Reply:
this is included in the list above.